web-client/src/ustc-ui/Pagination/Paginator.tsx

```diff
@@ -1,51 +1,60 @@
+import { Button } from '@web-client/ustc-ui/Button/Button';
 import React from 'react';
-import ReactPaginate from 'react-paginate';
 
 export const Paginator = ({
-  breakClassName,
-  forcePage,
-  id,
-  marginPagesDisplayed = 3,
+  currentPageIndex,
   onPageChange,
-  pageCount,
-  pageRangeDisplayed,
+  totalPages,
 }: {
-  breakClassName?: string;
-  marginPagesDisplayed?: number;
-  pageCount: number;
-  id?: string;
-  pageRangeDisplayed: number;
-  onPageChange: (selectedItem: { selected: number }) => void;
-  forcePage: number;
+  totalPages: number;
+  currentPageIndex: number;
+  onPageChange: (currentPage) => void;
```

Comment:
Could we specify the expected type of the `currentPage` parameter in the `onPageChange` function?

Reply:
Good call, done!